lib/addons/opa-gatekeeper/index.ts

```diff
@@ -0,0 +1,53 @@
+import { ClusterAddOn, ClusterInfo } from "../../spi";
+
+/**
+ * Configuration options for the add-on.
+ */
+export interface OpaGatekeeperAddOnProps {
```

Comment:
You are no using any of the properties defined in the options. All of them must be used for provisioning.

Reply:
addressed